### ISSUE TITLE
Enable cluster_admin in Kubernetes service connections

### ISSUE THREAD

**Issue by Chathuru** [microsoft/terraform-provider-azuredevops#216]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description
- When creating Kubernetes service connections with Azure subscription authentication type from the UI, we have the option to allow the use of cluster admin credentials. Currently, as to my knowledge, we don't have an option in the azuredevops provider. I have partially done the changes and tests and it's working https://github.com/Chathuru/terraform-provider-azuredevops/commit/7c13b9663041bdb60dfdccb8e2d4756dee93ba21. It could be handy if this feature added.

- Creation of environments for adding manual approval.

### New or Affected Resource(s)

* azuredevops_serviceendpoint_kubernetes

### Potential Terraform Configuration

```hcl
resource "azuredevops_serviceendpoint_kubernetes" "kubernetes" {
  project_id            = data.azuredevops_project.p.name
  service_endpoint_name = "iam-cloud"
  apiserver_url         = "https://k8s.azmk8s.io"
  authorization_type    = "AzureSubscription"

  azure_subscription {
    subscription_id   = "xxx"
    subscription_name = "xxx"
    tenant_id         = "xxx"
    resourcegroup_id  = "xxx"
    cluster_name      = "xxx"
    **cluster_admin     = true**
  }
}
```

### References

- https://github.com/Chathuru/terraform-provider-azuredevops/commit/7c13b9663041bdb60dfdccb8e2d4756dee93ba21
- https://github.com/Chathuru/terraform-provider-azuredevops/commit/fb32f5541e09a5c03335a6e066ba403bbf69c84e

**Comment by xuzhang3** [comment]
@Chathuru  Can you submit your PR? I can help review and merge. You may need add acceptance test for new property configuration. You can reference [resource_serviceendpoint_kubernetes_test](https://github.com/microsoft/terraform-provider-azuredevops/blob/master/azuredevops/internal/acceptancetests/resource_serviceendpoint_kubernetes_test.go)

**Comment by Chathuru** [comment]
@xuzhang3 I am new to Go programming. I will try to write unit test and acceptance test and send a PR.

**Comment by Chathuru** [comment]
@xuzhang3 Manage to update the unit test. Run the acceptance test and seems like they are passing the test. Git doesn’t allow me to create a PR, please have a look.

**Comment by xuzhang3** [comment]
@Chathuru  You should be able submit the RP. If you still cannot submit the PR, please send me your branch link, make sure you have submit all your latest code to your branch.
1. Go to your fork repo
![image](https://user-images.githubusercontent.com/57888764/98888714-71bf6480-24d3-11eb-8522-47d5e8b7930a.png)

2. Select the Pull request tab.
![image](https://user-images.githubusercontent.com/57888764/98888772-8c91d900-24d3-11eb-9d72-7b22a99d7699.png)

3. Create a new PR with `Create pull request` button. Choose the branch you want to submit.
![image](https://user-images.githubusercontent.com/57888764/98888805-a03d3f80-24d3-11eb-860d-f9b82f959f0e.png)

**Comment by shwetanshu-cldcvr** [comment]
Any estimated time for release for this?

**Comment by xuzhang3** [comment]
@shwetanshu-cldcvr  This feature has been released in v0.1.1 https://registry.terraform.io/providers/microsoft/azuredevops/latest/docs/resources/serviceendpoint_kubernetes

**Comment by xuzhang3** [comment]
Close this issue as this issue has been fixed. Feel free to open another issue if you have questions.
